packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -983,10 +983,11 @@ class StripeHelper {
       },
     } = invoice;
     const { id: planId, nickname: planName } = plan;
+    const productMetadata = this.mergeMetadata(plan, abbrevProduct);
     const {
       emailIconURL: planEmailIconURL = '',
       downloadURL: planDownloadURL = '',
-    } = this.mergeMetadata(plan, abbrevProduct);
```

Comment:
These changes are basically passing along the merged product/plan metadata through to email senders, where the user's locale is available and we can select the product details appropriate to that locale (i.e. including the terms & privacy URLs)